healpy/pixelfunc.py

```diff
@@ -1365,7 +1373,7 @@ def get_interp_val(m, theta, phi, nest=False, lonlat=False):
     p = np.array(r[0:4])
     w = np.array(r[4:8])
     del r
-    return np.sum(m2[p] * w, 0)
+    return np.squeeze(np.sum(m[:, p] * w, axis=1))[()]
```

Comment:
why are you using `[()]`?

Reply:
ok, can you add a comment explaining that? it is obscure syntax

Reply:
Sure, I've now added that. The CI will need to be approved to run again, it looks like.